common.sh

```diff
@@ -19,34 +19,34 @@ function setup_docker_machine() {
 
         # Check if the docker machine exists already, create one if not.
         if [[ $? == 1 ]]; then
-            notice "No ${MACHINE} environment found"
+            echo "No ${MACHINE} environment found"
```

Comment:
Not a dependancy but just a clean way to represent -> notices, warnings and errors with different colours. All this functionality is missing. I'll raise a separate issue